api/rda.py

```diff
@@ -71,6 +72,30 @@ def wrapper(body, **kwargs):
     return wrapper
 
 
+def endpoints(plugin=None):
+    plugins_list = ["epos", "gbif", "digital_csic", "dspace7", "signposting"]
+    plugins_with_endpoint = []
+    links = []
+
+    for plug in plugins_list:
+        try:
+            config = configparser.ConfigParser()
+            config.read("plugins/" + plug + "/config.ini")
+            links.append(config["Generic"]["endpoint"])
+            plugins_with_endpoint.append(plug)
+        except:
+            print("No endpoint found for " + plug)
+    # Create a dict with all the found endpoints
+    enp = dict(zip(plugins_with_endpoint, links))
+    # If the plugin is given then only returns a message
+    if plugin:
+        try:
+            return enp[plugin]
+        except:
+            return "Input plugin not found"
```

Comment:
The method needs to return the HTTP code as well, such as:

```python
return endpoint, 200
```

Reply:
Done